napalm/__init__.py

```diff
@@ -26,6 +27,7 @@ def get_network_driver(vendor):
         'JUNOS': JunOSDriver,
         'JUNIPER': JunOSDriver,
         'FORTIOS': FortiOSDriver,
+#        'IBM': IBMDriver,
```

Comment:
Ok, great :) Library with my bug fix is available on https://github.com/kderynski/blade-netconf-python-client , any unix like system with python 2.7 should be enough. If I understand correctly you will add this info or should I do this ?